code/script.js

```diff
@@ -291,54 +282,44 @@ const selectQuestion = () => {
         category
       }
   } else if (category === 'hat') {
-    currentQuestion = {
-      attribute: 'hat',
-      value: true, 
-      category
+      currentQuestion = {
+        attribute: 'hat',
+        value: true, 
+        category
     } 
   } else if (category === 'glasses') {
-    currentQuestion = {
-      attribute: 'glasses',
-      value: true, 
-      category
-    }  
+      currentQuestion = {
+        attribute: 'glasses',
+        value: true, 
+        category
+      }  
```

Comment:
To make a bit less code, you can combine categories "glasses" and "hat" into single category called "accessories", and then : 
`currentQuestion = {
      attribute: value,
      // 👆 this is the property of the booleans such as smoke, glasses and hat. add the value from the input here
      value: true, // we're asking if this person wears a hat for example, so always true in the question.
      category: category,
    }`